pkg/api/types.go

```diff
@@ -62,6 +62,9 @@ type Config struct {
 	// variables.
 	EnvironmentFile string
 
+	// BuildNamespace provides the namespace under which the labels will be generated.
+	BuildNamespaceLabel string
```

Comment:
If this was about cars and colors, we'd have:

```go
type ManufactureOrder struct {
   CarMake string
   CarColor string
   // ...
}

mo := ManufactureOrder{"Škoda", "blue", ...}
```

`CarColor`, not `ColorCar`, right?